### DISCUSSION TITLE
Another Panel in the WF Area

### DISCUSSION THREAD

**Discussion by GreyHound55** [comfyanonymous/ComfyUI#5154] [General]
Updated ComfyUI yesterday, and a new "fixed" Panel (zoom, reset, etc...) appeared. Just like a few days ago, when the Queue fixed-panel appeared, a day later it was made "repositionable". Thank you!

Can this new Panel, please, also be made "repositionable" ?

I can't stress enough (like before), how the WF Area is at a premium and these Panels should not be stealing space from the WF Area. These Control Panels are secondary at best, in terms of priority. WF Area is where 99% of the focus/work occurs !

I'm still scratching my head after the last time ... Why the person who developes these, thinks this is a good idea ??? Almost like piecemeal development.

**Comment by ltdrdata** [comment]
Please move this request to front repo.
https://github.com/Comfy-Org/ComfyUI_frontend
